app/models/incomeSourceDetails/viewmodels/CheckDetailsViewModel.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.incomeSourceDetails.viewmodels
+
+import enums.IncomeSourceJourney.IncomeSourceType
+
+import java.time.LocalDate
+
+
+case class CheckDetailsViewModel(businessName: Option[String] = None,
```

Comment:
based on this model definition its just 3 field would be mandatory, can you confirm that this is correct?